internal/events/trigger/forward_event.go

```diff
@@ -96,13 +97,13 @@ func ForwardEvent(p ForwardParamters) (*http.Response, error) {
 	return resp, nil
 }
 
-func getSignatureHeader(req *http.Request, id string, secret string, transport string, payload []byte) {
+func getSignatureHeader(req *http.Request, id string, secret string, transport string, timestamp string, payload []byte) {
 	mac := hmac.New(sha256.New, []byte(secret))
 	ts := util.GetTimestamp()
 	switch transport {
 	case models.TransportEventSub:
-		req.Header.Set("Twitch-Eventsub-Message-Timestamp", ts.Format(time.RFC3339Nano))
-		prefix := ts.AppendFormat([]byte(id), time.RFC3339Nano)
+		req.Header.Set("Twitch-Eventsub-Message-Timestamp", timestamp)
+		prefix := ts.AppendFormat([]byte(id), timestamp)
```

Comment:
This is suspect to me. `AppendFormat` adds a prefix (in this case the ID) and a format to a time. You'll want to do: 
```suggestion
		req.Header.Set("Twitch-Eventsub-Message-Timestamp", timestamp)
		prefix := timestamp.AppendFormat([]byte(id), time.RFC3339Nano)
```

And nuke the `ts` var since you're leveraging the existing timestamp parameter.